metabase/metabase
# Add timeline editing

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Shows events in a sorted list
- Adds basic timeline & events editing

How to test:
- Switch to `events-migration` branch, then run `yarn dev`
- When Metabase is ready, switch to this branch without restarting the backend
- Go to a collection and click on `Events` icon
- Add a timeline and a few events
- Make sure events are displayed in a sorted list
- Make sure it is possible to edit timelines and events via the menu

<img width="804" alt="Screenshot 2022-02-14 at 17 36 42" src="https://user-images.githubusercontent.com/8542534/153884206-392c8fbe-bac6-4265-8142-7e9a3f1a428e.png">

<img width="838" alt="Screenshot 2022-02-14 at 17 35 02" src="https://user-images.githubusercontent.com/8542534/153883940-6f97e344-c9ac-415c-baa5-5f99bab8b740.png">
